app/views/layouts/application.html.haml

```diff
@@ -16,6 +16,7 @@
       = link_to image_tag('logo.png'), 'https://www.magmalabs.io', title: 'TIL MagmaLabs', class: 'ml'
       = link_to root_url do
         %h1 TODAY I LEARNED
+      = render 'shared/google_analytics'
```

Comment:
no need to update but I do prefer use the view inheritance. Instead of creating a `shared` folder and put there anything that will be used accros the `application`, I would simply put the file inside `views/application` folder

`app/views/application/_google_analytics.html.haml`

so you will render:

```
= render 'google_analytics'
```